src/tagging/sass/multiSelectFix.scss

```diff
@@ -0,0 +1,6 @@
+.react-select__value-container.react-select__value-container--is-multi {
+  &.react-select__value-container--has-value {
+    padding-right: 22px !important;
```

Comment:
Isn't there a way how to make this rule more specific so we don't have to use `!important`? If not leave it be, but it would be nicer without it.